app.py

```diff
@@ -1404,7 +1404,7 @@ def index(level, program_id):
     max_level = hedy.HEDY_MAX_LEVEL
     level_number = int(level)
     return render_template(
-        "hedy-page/code-page.html",
```

Comment:
This will cause problems in many other pages, the mentions to this template are not accidental, they are intentional. The reason for the duplciated code-page.html template is due to the public adventures still needing the old one.

Reply:
I think the problem in the new template is that is missing a variable, can you try fixing it instead?